Templates/DefaultComponent/Template/Include/${Name}Interface.h

```diff
@@ -0,0 +1,48 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace ${SanitizedCppName}
+{
+    class ${SanitizedCppName}Requests
```

Comment:
Does this mean that the name of the component is also the namespace name?

Reply:
That is a good question. Is there a way to specify the namespace separately from the component name? Assuming this was going into a gem, the gem name would normally be the namespace.

Reply:
There is a way with the replace option that the create-from-template command supports, such that the GemName can be substituted whenver there is a ${GemName} placeholder.
https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/engine_template.py#L2387-L2394

Reply:
I've added a ${GemName} placeholder variable and will update the command line example in the PR description to match.
I've also added this placeholder to the Include path for the bus interface file, since this is part of the default path.